README.md

```diff
@@ -64,30 +66,32 @@ The following file types are detected automatically:
 
 To use it with GoHugo and basic `.html` files, you'll have to override the used parser inside your `.prettierrc` file:
 
-```js
+```json
 {
-  overrides: [
+  "overrides": [
     {
-      files: ["*.html"],
-      options: {
-        parser: "go-template",
-      },
-    },
-  ],
+      "files": ["*.html"],
+      "options": {
+        "parser": "go-template"
+      }
+    }
+  ]
 }
+
 ```
 
 ## VSCode
 
-Make sure to always have installed **both** dependencies:
-
-- `prettier`
-- `prettier-plugin-go-template`
+Steps:
 
-Also make sure that they are installed inside the same scope.
-Install both globally (`npm i -g`) or locally – otherwise prettier may not pick up the plugin.
+0. Have node & npm installed
```

Comment:
I wonder if we can have simpler instructions like https://github.com/withastro/prettier-plugin-astro#using-in-vs-code ? I haven't needed to do anything like the instructions in this PR to get prettier to work in my own projects

Reply:
There are basically two use-case:
- Globally installed: Hard & complex to setup
- Locally installed: Easy just like the astro guide you just pointed to (this is also what you're probably using @jasikpark )

I'd be strongly for recommending the local use, but there are quite a few people who don't want to have the dependency locally but instead "just-a-formatter".